js/mapbox-gl-js-common.js

```diff
@@ -2,101 +2,111 @@
 
 var defaultStyle = {};
 
+loadLayers();
+function loadLayers() {
```

Comment:
I really like the layers in separate files.

However, I wonder if this causes potential weird race conditions since we're loading these async and may get into situations where we're trying to access `MAP_LAYERS` or `MAP_LOCATIONS` before they have been loaded.

One other way to do this would be to make `mapLayers.json` a `.js` file instead and just include it with a `<script>` tag and make it like `MAP_LAYERS = [...];` -- not sure if this is actually a concern.